.cargo/config.toml

```diff
@@ -13,4 +13,9 @@ rustflags = [
     "-Wclippy::print_stderr",
     "-Wclippy::implicit_clone",
     "-Aclippy::items_after_test_module",
+
+    # Strange clippy warnings when upgrading rust toolchain to nightly-2023-07-30.
+    # It will complain if the trait implementor has functions with `unimplemented!()` in it.
+    # Allow it for now; try disallow it when the toolchain is upgraded in the future.
+    "-Aclippy::diverging_sub_expression",
```

Comment:
Does an elder nightly version also have this error?

Reply:
Don't know which version introduces this error, maybe some version in june.

Reply:
update the rust again and this false positive is gone, PTAL @evenyag